CIMonitor/CIMonitor
# Switch from Yarn to NPM and update the dependencies

### What

- Removed Yarn and switched to NPM instead
- Updated all NPM packages as far as possible
- Disable Vue styling rule that conflicted with Prettier

### Why

Gotta keep up with the times man. Also NPM is not that bad anymore so it can replace Yarn again.

### How to test

1. Pull this PR
1. Delete the node_modules directory
1. Run `npm install`
1. Run the local dev-server, dev-module-client, and dev-dashboard to see everything works 
1. Run the local tests to see they still pass

### Todo

- [ ] Find a solution for [Laravel Mix' deep support](https://github.com/JeffreyWay/laravel-mix/issues/1850) and rewrite the code to solve it
- [ ] After [this PR](https://github.com/jonschlinkert/cache-base/pull/12) is merged, run `npm install` and `npm audit` again to see if it solves the audit warnings
